.github/actions/cypress-e2e-testing/action.yml

```diff
@@ -26,20 +28,20 @@ runs:
     - name: "Start Centreon container"
       run: docker run -p 4000:80 -d -v "/var/run/docker.sock:/var/run/docker.sock" --name e2e-tests-${{ inputs.module }} ${{ inputs.image }}:${{ inputs.image_version }}
       shell: bash
-
     - name: Cypress end-to-end testing
       uses: cypress-io/github-action@v4
       with:
         browser: chrome
         install: false
         working-directory: ${{ inputs.module }}/tests/e2e
+        spec: cypress/e2e/**/${{ inputs.path_spec }}
 
     - name: Archive videos
       uses: actions/upload-artifact@v3
       if: ${{ failure() }}
       with:
         name: cypress-videos
-        path: ${{ inputs.module }}/tests/e2e/cypress/results/videos
+        path: ${{ inputs.module }}/tests/e2e/cypress/results/videos/${{ inputs.path_spec }}/
```

Comment:
I meant about the name property rather than the path :)

Reply:
Like this for example: cypress-videos-${{ inputs.path_spec }} ?